fileonfeature.txt

```diff
@@ -1,3 +1,4 @@
 this is my file on feature branch
 
 
+dodaje kolejną linikję tekstu
```

Comment:
This is not the way how we add new lines to our txt file.
It should start with "added" for e.g.